src/Workspaces/Core/Portable/Shared/Extensions/IFindReferencesResultExtensions.cs

```diff
@@ -48,11 +50,23 @@ public static bool ShouldShow(this ReferencedSymbol referencedSymbol)
                 return false;
             }
 
-            // We don't want to clutter the UI with property accessors if there are no direct
+            // If we're associating property references with an accessor, then we don't want to show
+            // a property if it is has no references.  Similarly, if we're associated associating
+            // everything with the property, then we don't want to include accessors if there are no
             // references to them.
-            if (definition.IsPropertyAccessor())
+            if (options.AssociatePropertyReferencesWithSpecificAccessor)
             {
-                return false;
+                if (definition.Kind == SymbolKind.Property)
+                {
+                    return false;
+                }
+            }
+            else
```

Comment:
`else if`?

Reply:
intentional in this form.  it helps keep both sides of the topmost 'if' symettrical.